src/app/language-support/page.tsx

```diff
@@ -1,17 +1,21 @@
 'use client';
 
-import { getAllLanguageSupport } from '@/api/supabase/queries/language-support';
+import { getAllInterpretation } from '@/api/supabase/queries/interpretation';
 import { useEffect, useState } from 'react';
-import { LanguageSupport } from '@/types/schema';
+import { DocumentTranslation, Interpretation } from '@/types/schema';
+import { getAllDocuments } from '@/api/supabase/queries/documentTranslation';
```

Comment:
nitpick: can we move the queries to consecutive lines? or i can change this next time 
i.e. move line 6 to after line 3